apps/emqx/test/emqx_bpapi_SUITE.erl

```diff
@@ -0,0 +1,62 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_bpapi_SUITE).
+
+-compile(export_all).
+-compile(nowarn_export_all).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+-include_lib("emqx/src/bpapi/emqx_bpapi.hrl").
+
+all() -> emqx_common_test_helpers:all(?MODULE).
+
+init_per_suite(Config) ->
+    emqx_common_test_helpers:start_apps([emqx]),
+    [mnesia:dirty_write(Rec) || Rec <- fake_records()],
+    Config.
+
+end_per_suite(_Config) ->
+    meck:unload(),
+    [mnesia:dirty_delete({?TAB, Key}) || #?TAB{key = Key} <- fake_records()],
+    emqx_bpapi:announce(emqx),
+    ok.
+
+t_max_supported_version(_Config) ->
+    ?assertMatch(3, emqx_bpapi:supported_version('i-dont-exist2@localhost', api2)),
+    ?assertMatch(2, emqx_bpapi:supported_version(api2)).
```

Comment:
Suggestion: maybe assert that the function raises when checking the version of a node that is not (yet?) in the table?